src/index.js

```diff
@@ -1,19 +1,48 @@
-import './css/base.scss';
-import { isValidTraveler, notValidTraveler } from './travel_tracker_service.js'
-import { isValidAgency, renderSuccessfulAgencyLogin, renderSuccessfulTravelerLogin, loginError, isValidPassword } from './login_helper.js';
+import "./css/base.scss";
+import {
+  isValidTraveler,
+  getSingleTraveler,
+  getTravelerTrips,
+  getTravelerDestinations,
+  getTravelerDashboardData,
+  getApprovedTripsForAgent,
+  getDestinationsForAgent,
+} from "./travel_tracker_service.js";
+import {
+  isValidAgency,
+  renderSuccessfulAgencyLogin,
+  renderTravelerDashboard,
+  loginError,
+  isValidPassword,
+} from "./login_helper.js";
 
-let button = document.getElementById('submit');
+let button = document.getElementById("submit");
 
 button.onclick = function processLogin() {
-  let usernameInput = document.getElementById('input-username').value;
-  let passwordInput = document.getElementById('input-password').value;
-  let validTraveler = isValidTraveler(usernameInput);
+  let usernameInput = document.getElementById("input-username").value;
+  let passwordInput = document.getElementById("input-password").value;
   if (isValidPassword(passwordInput)) {
     isValidTraveler(usernameInput).then(function (result) {
       if (result === true) {
-        renderSuccessfulTravelerLogin();
-      } else if
-        (isValidAgency(usernameInput)) {
+        var travelerId = usernameInput.slice(8);
+        getSingleTraveler(travelerId).then(function (result) {
+          let traveler = result.traveler;
+          getTravelerTrips(traveler).then(function (result) {
+            let travelerTrips = result.travelerTrips;
+            getTravelerDestinations(travelerTrips, traveler).then(function (
+              result
+            ) {
+              var destinations = result.travelerDestinations;
+              var trips = result.travelerTrips;
+              getTravelerDashboardData(traveler, destinations, trips).then(
```

Comment:
After looking at this actual function `getTravelerDashboardData`, I realized it isn't making a fetch call.  I am pretty sure this means it can be a regular heloer function and not a `.then` call

So 
```
let travelerDashboardData = prepareTravelerDashboardData(traveler, destinations, trips) // notice i renamed the function
renderTravelerDashboard(travelerDashboardData);
```

Reply:
has been moved